src/test/java/org/example/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package org.example;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
I can see that unit tests are not your favourite :D But they are super nice to implement!
We can talk on Monday or some other day if you want to try to write some unit tests for your apps ;)